tests/integration/cocoon/test_cocoon_transactions.py

```diff
@@ -166,6 +199,7 @@ def test_load_from_data_frame_transactions_success(
             identifier_mapping=identifier_mapping,
             property_columns=property_columns,
             properties_scope=properties_scope,
+            batch_size=batch_size
         )
 
         self.assertTrue(
```

Comment:
Just looking at the assert, does this pick up failures correctly i.e. when the request fails do we get back an empty (valid) version which makes this pass or do we not get that back which means what we have is currently ok?

Reply:
On a failure we get an exception back.

Reply:
However this only checks successes and you are correct that we need to check that the number of successes is greater than 0.